src/vm/docs/mod.rs

```diff
@@ -448,12 +448,14 @@ inputted value.",
 };
 
 const MAP_API: SpecialAPI = SpecialAPI {
-    input_type: "Function(A) -> B, (list A)",
-    output_type: "(list B)",
-    signature: "(map func list)",
+    input_type: "Function(A) -> B, (list A), (list B), ...",
```

Comment:
Shouldn't this be:
```
input_type: "Function(A1, A2, ..., An) -> B, (list A1), (list A2), ..., (list An)",
output_type: "(list B)",
signature: "(map func list-A1 list-A2, ..., list-An)",
```

The function doesn't have to return type `B` if the third argument is a `(list B)`, does it?

Reply:
Shouldn' this refer to a _sequence_ rather than a list?
Also: https://github.com/clarity-lang/reference/issues/7

Reply:
Is
```
    input_type: "Function(A, B, ..., N) -> X, (list A1 A2 ... Am), (list B1 B2 ... Bm), ..., (list N1 N2 ... Nm)",
    output_type: "(list X)",
    signature: "(map func list-A list-B ... list-N)",
```
maybe more clear?

Reply:
> maybe more clear?

The `map` function can also take other sequence types (buffers and strings), so shouldn't the signature be more like:

```clojure
(map func seq-A seq-B ... seq-N) 
```